### ISSUE TITLE
Spell Checking Keybindings

### ISSUE THREAD

**Issue by duranivan** [NeoVintageous/NeoVintageous#599]
While having Sublime's spell checking on, I'm wondering if the keybindings mentioned in `:help spell` are supposed to work. None of them work for me, i.e. `[s`, `[s`, `z=`, `zg`, etc...

Other keybindings, for example the `unimpaired` ones `[t`, `]b`, `[<space>`, etc., do work for me, so I doubt some other package or keybindings are causing a conflict.

So I don't know if I am missing something, or if I should have something activated for these keybindings to work...

When I use unimpaired command `[<space>`, for example, this is I get in the console:

```
INFO:NeoVintageous.nv.commands:key evt: [ repeat_count=None do_eval=True check_user_mappings=True
INFO:NeoVintageous.nv.commands:key evt: <space> repeat_count=None do_eval=True check_user_mappings=True
INFO:NeoVintageous.nv.vim:command (action): {'action': '_nv_unimpaired', 'action_args': {'action': 'blank_up', 'mode': 'mode_internal_normal', 'count': 1}}
```

However, if I use, say, `z=` for spell checking options, this is what I get:

```
INFO:NeoVintageous.nv.commands:key evt: z repeat_count=None do_eval=True check_user_mappings=True
INFO:NeoVintageous.nv.commands:key evt: = repeat_count=None do_eval=True check_user_mappings=True
```

So basically it seems like no command is being activated.

Any feedback is appreciated!

I'm running Mac OS 10.14.5 and Sublime Text Build 3208

**Comment by gerardroche** [comment]
These will be new commands. Some are easy and will be implemented in the next version, I'm not sure yet about some others:

* [x] `[s` (next version 1.7.0)
* [x] `]s` (next version 1.7.0)
* [ ] `z=`
* [ ] `zg` 
* ...

**Comment by gerardroche** [comment]
fyi you can use the unimpared command `cos` to toggle spell checking. 

```

Option toggling.

 On      Off    Toggle  Option
*[oa*   *]oa*   *coa*   'menu'          {NeoVintageous specific}
*[oc*   *]oc*   *coc*   'cursorline'
*[oe*   *]oe*   *coe*   'statusbar'     {NeoVintageous specific}
*[ol*   *]ol*   *col*   'list'
*[om*   *]om*   *com*   'minimap'       {NeoVintageous specific}
*[on*   *]on*   *con*   'number'
*[ot*   *]ot*   *cot*   'sidebar'       {NeoVintageous specific}
*[os*   *]os*   *cos*   'spell'
*[ow*   *]ow*   *cow*   'wrap'
```

Note it looks like unimpaired has changed the toggle mappings a few times since I implemented them The default is now `yo` instead of `co`. Previously `co` was changed to `=o`. I think they've settled on `yo` so I'll try implement that alongside the legacy `co` toggles in the next version.

**Comment by duranivan** [comment]
Thanks @gerardroche, I will certainly try those commands out.
At the moment, as a replacement for `z=` I have this keybinding, however, for some reason, many times it doesn't show me the list of right words suggestions (only if I right click with the mouse):

```
  { "keys": ["command+alt+."], "command": "context_menu" },
```

Looking forward to having the `z=`'s commands working in Neovintageous.

Thanks again!

**Comment by gerardroche** [comment]
I'm not sure if we'll be able to implement `z=` particularly well. it looks like sublime passes some coordinates to the context menu command when right clicked so we might be able to mimic that functionality. 

```
command: context_menu {"event": {"button": 2, "x": 837.062852648, "y": 86.0042317708}}
```

I've asked on the forum if anyone has more info on the spelling related commands. https://forum.sublimetext.com/t/spell-checking-apis/44962/2

It looks like I can implement the `wrapscan` option too and the related ex mode commands  too.

**Comment by gerardroche** [comment]
Here's what has been implemented:

* [x] `[s`
* [x] `]s`
* [x] `:set spell` and `:set nospell` (next version 1.7.0); depends on #585
* [x] `:set spell!` and `:set spell?` (next version 1.7.0); depends on #585
* [x] `'wrapscan'` option including `:set wrapscan`, `:set nowrapscan`, `:set ws`, `:set nows`, etc.
* [x] `z=`
* [x] `zg`
* [x] `zug`
* [x] `:spellgood {word}`
* [x] `:spellundo {word}`  

All will be in the next version, 1.17.0.
